graphics/fonts/bdf.cpp

```diff
@@ -742,6 +742,9 @@ BdfFont *BdfFont::scaleFont(const BdfFont *src, int newSize) {
 		return nullptr;
 	}
 
+	BdfFont *srcCopy = new BdfFont(src->_data, DisposeAfterUse::YES);
+	srcCopy->initializeScaling(src->getFontSize(), newSize);
```

Comment:
initializeScaling() should be called for this copy of the object, not creating another one pointlessly.